rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-retrieve-results-from-mulesoft-idp.adoc

```diff
@@ -0,0 +1,54 @@
+= Retrieve Results from MuleSoft IDP 
+
+Retrieve the results of a document processed with MuleSoft IDP. To run this action step you must provide an Execution ID, which generates during the execution of the Submit Document to MuleSoft IDP action step. The Execution ID links this action step with a document action execution to define which results to retrieve. 
+
+include::partial$mulesoft-idp.adoc[tag=creditsWarning]
+
+== Before You Begin
+
+Before you begin using this action step, ensure that: 
+
+* Your Anypoint user account has the *Build Actions* permission
+* You have the Execution ID generated from a Submit Document to MuleSoft IDP action step. 
+* You have IDP document actions published in your organization
++
+See xref:idp::publishing-document-actions.adoc[] for additional information. 
+* You have a Connected App configured to access IDP
++
+See xref:idp::automate-document-processing-with-rpa.adoc[] for configuration instructions. 
+
+== Properties
+
+This action step has no properties other than *Common* and *Timing*. See xref:common-properties.adoc[] for further details. 
+
+== Inbound Variables
+
+* *Client ID*
++
+Client ID of the Connected App configured to call IDP.
+* *Client Secret*
++
+Client Secret of the Connected App configured to call IDP.
+* *Connected App Credentials*
++
+Pin credentials containing the Client ID and Client Secret of the Connected App configured to call IDP.
+* *Execution ID*
++
+Execution ID of the document processing operation. 
+
+== Outbound Variables
+
+* *Status*
+* *JSON Result*
+
+[[wizard]]
+== Wizard
+
+Specify the *Execution ID* to query and the corresponding IDP *Action* and *Action Version*. These lists populate with the available document actions that your organization has published in IDP. 
```

Comment:
**Nitpick (non-blocking)**: I think we should emphasize that the Action and Version must exactly match the configuration of the Submit AS which generated the Execution ID.